ttnn/cpp/ttnn/operations/copy.hpp

```diff
@@ -49,6 +50,15 @@ struct Typecast {
                 output_dtype == optional_output_tensor.value().get_dtype(),
                 "If both output dtype and output tensor provided dtype should match");
         }
+        if (input.device()->arch() == tt::ARCH::GRAYSKULL) {
+            TT_FATAL(
```

Comment:
Should these fatals go to the experimental op?

Reply:
Yeah better to put them in the validates